src/coreclr/inc/CrstTypes.def

```diff
@@ -577,3 +577,7 @@ End
 Crst PgoData
     AcquiredBefore LoaderHeap
 End
+
+Crst StaticBoxInit
+    Unordered
```

Comment:
This should have proper level assigned.

`Unordered` is meant for very special situations only and it needs a very long comment explaining why it is ok.

Reply:
Thanks, I'm a bit puzzled on what should I specify here - I came up with `AcquiredBefore LoaderHeap FrozenObjectHeap`

My understanding that the function is either called from jitinterface.cpp (no top-level locks?) -- `SameLevelAs Jit` ?
Or from runtime DoInitClass - `AcquiredAfter ClassInit` ?

Reply:
`AcquiredBefore LoaderHeap FrozenObjectHeap` should work fine